core/modules/system/system.module

```diff
@@ -179,6 +179,10 @@ function system_permission() {
     'block IP addresses' => array(
       'title' => t('Block IP addresses'),
     ),
+    'flush caches' => array(
+      'title' => t('Flush caches'),
+      'description' => t('Access links to flush caches.'),
```

Comment:
These aren't always links anymore. Maybe 'links and buttons' or maybe we leave that bit out?

Reply:
How about 'Provides access to flush caches from the user interface'?

Reply:
Maybe `manually`?  `Provides access to manually flush the caches from the user interface` or even `Provides access to manually flush the caches.`